### ISSUE TITLE
look_at gives strange results

### ISSUE THREAD

**Issue by tobiblender** [godotengine/godot#35880]
**Godot version:**
3.2
**OS/device including version:**
win8
**Issue description:**
i place skidmarks on terrain via the code
```gdscript
var new_skidmark
new_skidmark = skidmark.instance()				

# add skidmark
$"../..".add_child(new_skidmark)

# get Raycast Normal + HitPoint
var normal = $RayCast.get_collision_normal()
var point = $RayCast.get_collision_point()

# move skidmark to hitpoint
new_skidmark.translation = point

# hitpoint/hitnormal/velocity vector transform via look_at
# HERES THE PROBLEM since 3.2 !!!!!!!!!!!
new_skidmark.look_at(point - normal, get_linear_velocity() )

# move skidmark away from ground a little
new_skidmark.translate_object_local(Vector3(0,0,0.08))
```
now the problem/bug since 3.2:
if the ground is totally flat - the skidmarks are NOT rotated correctly along the velocity vector.
if the ground is just a little bit uneven, it works fine.

this look_at code **worked in 3.1.2 just fine** - in every case.
since 3.2 theres that problem if ground is totally even

look at hit_normal UP-AXIS=velocity_vector should work - and did 3.1.2

**Steps to reproduce:**
- open the attached project
- open the scene "look_at_bug"
- press F5 and see the car driving and the skidmarks all over the place (rotation-wise)
- now rotate the "GROUND" Meshs X-Axis 1 degree   (same scene)
- press F5 and see the skidmark-rotation working just fine

**Minimal reproduction project:**
[look_at_bug_1.zip](https://github.com/godotengine/godot/files/4148680/look_at_bug_1.zip)

**Comment by tobiblender** [comment]
sorry for the strange format of my code
i used the code tag to insert the code in the issue... it looks totally broken... and i cant edit the issues content... just the title... is that normal?

**Comment by Zireael07** [comment]
Press three dots at the top of your post and you should be able to edit it.
The code tag is sometimes finicky in my experience.

**Comment by KoBeWi** [comment]
The default code tag is for single lines. Use ``` instead, like
\`\`\`
code
\`\`\`

**Comment by groud** [comment]
Please provide a minimal reproduction project. It is needed for the issue to be solved.

**Comment by tobiblender** [comment]
Hello!  :)
okay i attached a mini-project to the mein thread.
thx for helping :)

**Comment by weckr** [comment]
I think I have related issue, since i also have perfectly perpendicular inputs for look_at, tried to upgrade my project from 3.1 to 3.2.

look_at( (0,0,1) , (1,0,0) ) 
works the same but if i call it on the same object but with
look_at( (0,-1,0) , (1,0,0) )
the object is flipped in the up axis (1,0,0) which was not the case in the previous version and it breaks a big chunk of my project.

ive seen some changes to look_at to fix its issue with scaling in 3.2, but it might had broken it.

**Comment by kleonc** [comment]
Seems like this is already fixed. Just tested provided minimal reproduction project and:

- `3.2.stable` - [bug](https://user-images.githubusercontent.com/9283098/120357609-eccc5900-c305-11eb-879c-1617d76a856e.png),
- `3.2.3.stable` - [no bug](https://user-images.githubusercontent.com/9283098/120357991-50ef1d00-c306-11eb-806e-651b4a507ed8.png),
- `3.3.2.stable` - [no bug](https://user-images.githubusercontent.com/9283098/120357687-02418300-c306-11eb-9388-cd16d8e52fb7.png).
